docs/modelserving/data_plane/v1_protocol.md

```diff
@@ -0,0 +1,26 @@
+# Data Plane (V1)
+KServe's V1 protocol offers a standardized prediction workflow across all model frameworks. This protocol version is still supported, but it is recommended that users migrate to the [V2 protocol](TODO: LINK). 
```

Comment:
We recommend migrating to v2 inference protocol for performance and standardization among serving runtimes, however depending on your specific use cases, if you want a more flexible schema v1 can still be an option.

Reply:
done